packages/features/users/components/UserTable/UserListTable.tsx

```diff
@@ -204,12 +204,15 @@ export function UserListTable() {
         id: "teams",
         header: "Teams",
         cell: ({ row }) => {
-          const { teams, accepted } = row.original;
+          const { teams, accepted, email } = row.original;
           // TODO: Implement click to filter
           return (
             <div className="flex h-full flex-wrap items-center gap-2">
               {accepted ? null : (
-                <Badge variant="red" className="text-xs">
+                <Badge
+                  variant="red"
+                  className="text-xs"
+                  data-testid={`email-${email.replace("@", "")}-pending`}>
```

Comment:
To be able to check if the invited user is verified either after they sign up or because the org is verified in which case the user is automatically accepted.